packages/server/database/migrations/20200828161500-samlDomainAsId.ts

```diff
@@ -0,0 +1,26 @@
+import {R} from 'rethinkdb-ts'
+export const up = async function(r: R) {
+  try {
+    await r
+      .table('SAML')
+      .update((row) => ({
+        id: row('domain')
+      }))
+      .run()
+  } catch (e) {
+    console.log(e)
+  }
+  try {
+    await r
+      .table('SAML')
+      .filter((row) => row('id').ne(row('domain')))
```

Comment:
we use the field `domain` so let's keep it for now. IMO a little duplicate data is worth the expressiveness

Reply:
Doesn't this only remove the rows where the ID !== domain? It doesn't remove the domain field?

Reply:
ahh i think i imagined a `.without`. you're right